app/components/App.js

```diff
@@ -29,6 +47,13 @@ class App extends Component {
           color="#841584"
           accessibilityLabel="About"
         />
+        <Button
+          onPress={() => this.triggerLogin()}
+          title="Login with Trakt"
+          color="#841584"
```

Comment:
I see that this color `#841584` it's in more than one place.
Not required for this issue but maybe will be good to create a `color-constant.js` or similar in case we want to change the color scheme we can do it in one place